README.md

```diff
@@ -212,7 +175,6 @@ You can customize the settings in gist settings like:
     "sync.forceDownload": false,
     "sync.forceUpload": false,
     "sync.quietSync": false,
-    "sync.askGistName": false,

```

Comment:
Why is this removed?
If its `enabled` it asks for GIST Name.

Reply:
This setting exists in `CustomConfig`, not `ExtensionConfig`. Therefore having this setting mentioned in the `README.md` could mislead people.

Reply:
Nice catch, thanks